clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2420,11 +2420,11 @@ command line.
 
 The current set of options correspond to the existing "subtarget features" for
 the target with or without a "-mno-" in front corresponding to the absence
-of the feature, as well as ``arch="CPU"`` which will change the default "CPU"
-for the function.
+of the feature, as well as ``arch="CPU"`` and ``cpu="CPU"`` which will change
+the default "CPU" for the function.
```

Comment:
These docs confuse me a bit because I'm left wondering what the difference is between using `-arch="whatever"` and `-cpu="whatever"`; it sounds like they both do the same thing. If they indeed are identical, we should document that more clearly (but it certainly begs the question of "why do we have both?").

Reply:
I have some notes at https://maskray.me/blog/2022-08-28-march-mcpu-mtune . It seems that x86 uses `-march=` for what some other architectures (aarch64,arm,riscv,etc) use for `-mcpu=`...

Reply:
Thanks for explanation. Then x86 should be the special one, not aarch64.